src/libraries/System.Numerics.Vectors/tests/Matrix4x4Tests.cs

```diff
@@ -868,30 +867,142 @@ public void Matrix4x4CreateLookAtTest()
             Vector3 cameraUpVector = new Vector3(0.0f, 1.0f, 0.0f);
 
             Matrix4x4 expected = new Matrix4x4();
-            expected.M11 = 0.979457f;
-            expected.M12 = -0.0928267762f;
-            expected.M13 = 0.179017f;
```

Comment:
There's no change in these values, they've just been simplified to the shortest roundtrippable string for consistency